nexus/src/test/java/com/example/nexus/service/ProfileServiceImplTests.java

```diff
@@ -16,7 +16,6 @@
 @ExtendWith(MockitoExtension.class)
 class ProfileServiceImplTests {
     private static Profile profile;
-    private static AddMoneyRequest addMoneyRequest;
```

Comment:
why remove it

Reply:
This created a warning that this declaration should be local and not global, so it was moved in the addMoney test.